lib/handlers/allow.js

```diff
@@ -43,6 +43,10 @@ function allow (mode, checkPermissionsForDirectory) {
       resourcePath += '/'
     }
 
+    let trustedOrigins = [ldp.resourceMapper.resolveUrl(req.hostname)].concat(ldp.trustedOrigins)
+    if (ldp.multiuser) {
```

Comment:
Shouldn't we also include `serverUri` as a trusted origin in single-user mode?

Reply:
Yeah, so the `resolveUrl(req.hostname)` would then be that... So, you could turn this patch around, actually, with the same result. I think.